src/commands/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./config.command.ts";
```

Comment:
suggestion: there is not a huge advantage in index files in deno, and actually in esm, because you still have to import by full path (of the index file), let's try without index files and see if we actually need them